internal/jujuclistore/store.go

```diff
@@ -31,11 +31,11 @@ import (
 const (
 	timeoutRequestDuration = 5 * time.Minute
 
-	// launchPadURL is the base URL for the Juju binary downloads.
-	launchPadURL = "https://launchpad.net/juju"
+	// githubReleaseURL is the base URL for the Juju binary downloads.
+	githubReleaseURL = "https://github.com/juju/juju/releases/download/"
 
-	// launchPadTemplate is the template for constructing the download URL for Juju binaries.
-	launchPadTemplate = "{{.BaseURL}}/{{.VersionWithMinor}}/{{.VersionWithPatch}}/+download/juju-{{.VersionWithPatch}}-{{.Os}}-{{.Arch}}.tar.xz"
```

Comment:
The place where we format the template likely has an unused variable "VersionWithMinor" that can be cleaned up.

Reply:
https://github.com/canonical/jimm/pull/1716/files#diff-634c50b9219a62b8a03fdb9a2439bd20efb6078a97191b7de38ac4f574ad79aeL147